<?xml version="1.0"?>
<launch>
	<arg name="predict_time" default="1.5" />

	<arg name="backgroud" default="true" />
	<arg name="odom_topic" default="/bill" />
	<arg name="path_size" default="30" />

	<node name="leastsquare" pkg="human_motions" type="leastsquare_predict" output="screen">
		<param name="predict_time" value="$(arg predict_time)" />
	</node>

	<group if="$(arg backgroud)">
		<node name="pass_path" pkg="human_motions" type="pass_path" output="screen">
			<param name="odom_topic" value="$(arg odom_topic)" />
			<param name="path_size" value="$(arg path_size)" />
		</node>
		<node name="bag_player" pkg="rosbag" type="play" output="screen" args="-l --delay=1 --queue=10 $(find human_motions)/data/traj.bag" />
		<node name="rviz" pkg="rviz" type="rviz" output="screen" />
	</group>
</launch>
